cosigned/base/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    kubernetes.io/metadata.name: cosign-system
+  name: cosign-system
```

Comment:
not needed, as this was added [here](https://github.com/operate-first/apps/blob/73fe9f7d5c790f1b3521cda67963ef96b9f620f8/cluster-scope/base/core/namespaces/cosigned/namespace.yaml)